app/src/main/java/com/github/se/icebreakrr/MainActivity.kt

```diff
@@ -90,6 +91,7 @@ fun IcebreakrrApp() {
       composable(Screen.AROUND_YOU) {
         AroundYouScreen(navigationActions, profileViewModel, tagsViewModel)
       }
+      composable(Screen.OTHER_PROFILE_VIEW) { OtherProfileView(navigationActions) }
```

Comment:
Good idea i think, makes things clearer to differenciate your own profile to the other profiles